src/core/particle_data.cpp

```diff
@@ -598,6 +598,10 @@ int set_particle_q(int part, double q) {
   mpi_send_q(pnode, part, q);
   return ES_OK;
 }
+#ifndef ELECTROSTATICS
+const constexpr double ParticleProperties::q;
```

Comment:
you can drop the const here

Reply:
No, `const` is needed in C++11, `constexpr` only implies `const` in C++14 an newer.

Reply:
are you sure?
https://stackoverflow.com/questions/28845058/constexpr-const-vs-constexpr-variables